slices/libgdbm-compat4t64.yaml

```diff
@@ -0,0 +1,16 @@
+package: libgdbm-compat4t64
+
+essential:
+  - libgdbm-compat4t64_copyright
+
+slices:
+  copyright:
```

Comment:
nit: we typically put this slice at the bottom...just to be consistent with the other SDFs :p